client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js

```diff
@@ -38,12 +39,13 @@ function LaptopForm() {
 				lastName: lastName,
 				email: email,
 				phoneNumber: phoneNumber,
+				uuid: nanoid(10),
 			}),
 			headers: { "content-type": "application/json" },
 		})
 			.then((response) => response.json())
 			.then((data) => {
-				setRequestId(data.id);
+				setRequestId(data.uuid);
```

Comment:
in my imagined version of the system, this line does not change: the JSON that is sent back contains an id and the client doesn't really care what it is: it's just a number/string that gets passed around

Reply:
Done this